src/bundle/watch.ts

```diff
@@ -18,7 +18,12 @@ export const bundleAndWatch = async (options: BundleOptions) => {
   const watcher = new Watcher(options.rootDirectory, {
     recursive: true,
     ignoreInitial: true,
+    debounce: 0,
```

Comment:
by default this is 300ms, which was resulting in the bundler not being triggered before AVA re-ran tests

I think this should be ok (and it makes `edgespec dev` feel noticeably faster) but if it becomes an issue we could instruct users to have AVA ignore changes to their API route directory and instead watch the `.edgespec/` output folder (opposite of what happens now).

Reply:
hmmmm should be fine in 99% of cases, since users are unlikely to make a lot of changes to files really rapidly, right?

still, did you experiment with like 10ms or even 1ms instead? i could see this being really bad potentially if the user does like, `rm -rf node_modules`

Reply:
any introduced timing delay means a test may be able to make a request before the bundler has a chance to block, so imo let's leave it for now and re-evaluate if anyone's console gets spammed with build messages 😛 

what we really want is for the bundler to see the begining of the debounce period and block incoming requests *during* the debounce period

could PR an event for this to https://www.npmjs.com/package/watcher